src/presenters/PostReviewViewPresenter.vue

```diff
@@ -0,0 +1,33 @@
+<script>
+import PostReviewView from "../views/PostReviewView.vue";
+export default {
+  components: { PostReviewView },
+
+  props: {
+    model: Object,
+  },
+
+  methods: {
+    goToHomeACB() {
+    	this.$router.push("/");
+    },
+  },
+
+	data() {
+		return {
+			deckSize: this.model.getCurrentDeckSize(),
+			correctAnswersAmount: this.model.getCurrentDeckSize() - this.model.getWrongAnswers().length,
+		};
+	},
+};
+</script>
+
+<template>
+  <PostReviewView 
+	:currentDeckName="this.model.currentDeck.name"
```

Comment:
ta bort "this." från allting inom <template> tags (koden borde funka ändå)

Reply:
Oh in presenter too, I see